canonical/charmhub.io
# Move collaborators table to its own component

## Done
Added the bulk of the collaboration UI

## QA
- Go to https://charmhub-io-1414.demos.haus/CHARM_NAME/collaboration
- Check that there is a collaborators and invites table
- Check that the "Add collaborator" panel opens
- You should be able to add a collaborator and their email should show up in the invites table once you refresh the page

**Note**: The UI has some rough edges and the actions don't all work, collaborators are hardcoded. These things will be addressed in subsequent PRs.

## Issue
Fixes https://github.com/canonical/snapcraft.io/issues/4120
Fixes https://github.com/canonical/snapcraft.io/issues/4121
